lib/validation/validators.js

```diff
@@ -85,21 +85,27 @@ module.exports = Validators = {
   },
 
   // eslint-disable-next-line no-inline-comments, spaced-comment
-  before(value/*, [diff, unit][, diff, unit]*/) {
-    const args = [].slice.call(arguments, 1);
-    let date = moment(value, dateFormat);
-    let diff;
-    let unit;
-    while (args.length) {
-      diff = args.shift();
-      unit = args.shift() || 'years';
-      date = date.add(diff, unit);
+  before(value, date/*, [diff, unit][, diff, unit]*/) {
```

Comment:
To elaborate, any `arguments` property set on a validator gets `concat`-ed to the user input value and then used as the arguments to the validator function. So `arguments` will always* be an array, but unless there are nested arrays the actual arguments being passed into the validator function will not be arrays.

`*` I can't remember if single arguments are supported as not-arrays.